### ISSUE TITLE
not able to make ios /android builds

### ISSUE THREAD

**Issue by rohith-thummanapally** [RonRadtke/react-native-blob-util#198]
Hi , i was trying to make builds to upload the app to the app store and previously when i was trying to make the build which is on Nov 17, it was working fine , but now when i was trying to make the build it is giving error below , in the mean time i haven't touched any thing in the project

"While resolving: @config-plugins/react-native-blob-util@3.0.0 found : expo@45.0.8"

![image](https://user-images.githubusercontent.com/114573027/204765781-83f70117-a4b0-41fa-992e-2d9a5c7e6127.png)

* Expo sdk :- 45.0.0
* React-Native :- 0.68.2
* React-native-blob-util :- 0.16.2
* @config-plugins/react-native-blob-util :- 3.0.0
* I am using EAS.
* builds are not happening in  Android and Ios

**Comment by RonRadtke** [comment]
This library does not support expo. If you want to use expo together with this library you would have to check you project out.

**Comment by rohith-thummanapally** [comment]
but previously 2 weeks back everything was working fine, now it is showing this errors, if it doesn't work with expo i should get this errors previously also right? , btw i am not using plain expo i am using Expo Dev Client(EAS)

**Comment by RonRadtke** [comment]
It worked 2 weeks earlier with this lib?
The issue is, expo doesn't support libs with native code like mine does but EAS should take care of that - you're right.

It fails resolving config-plugins/react-native-blob-util :- 3.0.0
I honestly have no idea where this one is coming from - especially since 3.0.0 is not a valid version for the lib.

**Comment by rohith-thummanapally** [comment]
if 3.0.0 is not a valid version, then whats is the valid version for the lib?

**Comment by rohith-thummanapally** [comment]
i have installed this package in order to support for react-native-pdf according to this documentation https://github.com/expo/config-plugins/tree/main/packages/react-native-pdf#config-pluginsreact-native-pdf

can anyone help me, i was struck on this , i was not able to make any builds , and there is an update in the package14 days back, was that update making the issue?

**Comment by RonRadtke** [comment]
That looks fine. current version is 0.17.0.
The 3.0.0 refers to the config plugin. The current one would be 4.0.0 based on what I can see.
So technically your issue belungs here: https://github.com/expo/config-plugins/tree/main/packages/react-native-pdf

But based on the error - did you run the yarn / npm install for both packages correctly? 
Maybe you should also try an update of the packages since you have a conflicting peer dependenciesfor the expo version too.
But I can't fix this since it's a dependency of the config-plugin.
All of the expo scripts require expo ^47.0.3.

tldr: Issue is not in this library but with the config-plugin and comflicting peer dependencies. You will have to solve these issues before you can build sucessfully. Check your dependencies (expo version) and if required update your libraries. But I can't assist here since the porblem is not in this lib.

**Comment by rohith-thummanapally** [comment]
i have run npm install , will yarn install make a difference? 
and when i am trying to to yarn install , getting this" 'yarn' is not recognized as an internal or external command,"
and even  when i am trying to update the packages , nothing is working it is giving the same errors

**Comment by RonRadtke** [comment]
No shouöd not make any difference here if you're using npm or yarn.Then please create an issue for the expo/config since the problem is within this scope.Get Outlook for Android
